products/healthcare/terraform.yaml

```
@@ -0,0 +1,103 @@
+# Copyright 2019 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  Dataset: !ruby/object:Overrides::Terraform::ResourceOverride
+    description: |
+      {{description}}
+    id_format: "projects/{{project}}/locations/{{location}}/datasets/{{name}}"
+    import_format: ["projects/{{project}}/locations/{{location}}/datasets/{{name}}"]
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "healthcare_dataset_basic"
+        primary_resource_id: "default"
+        vars:
+          dataset_name: "example-dataset"
+          location: "us-central1"
+          time_zone: "America/New_York"
+    properties:
+      timeZone: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_from_api: true
+      creationTime: !ruby/object:Overrides::Terraform::PropertyOverride
+        exclude: true
+      selfLink: !ruby/object:Overrides::Terraform::PropertyOverride
+        ignore_read: true
+    custom_code: !ruby/object:Provider::Terraform::CustomCode
+      decoder: templates/terraform/decoders/long_name_to_self_link.go.erb
+  FhirStore: !ruby/object:Overrides::Terraform::ResourceOverride
+    description: |
+      {{description}}
+    id_format: "{{dataset}}/fhirStores/{{name}}"
+    import_format: ["{{dataset}}/fhirStores/{{name}}"]
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "healthcare_fhir_store_basic"
+        primary_resource_id: "default"
+        vars:
+          dataset_name: "example-dataset"
+          fhir_store_name: "example-fhir-store"
+          pubsub_topic: "fhir-notifications"
+    properties:
+      creationTime: !ruby/object:Overrides::Terraform::PropertyOverride
+        exclude: true
+      selfLink: !ruby/object:Overrides::Terraform::PropertyOverride
+        ignore_read: true
+    custom_code: !ruby/object:Provider::Terraform::CustomCode
+      decoder: templates/terraform/decoders/long_name_to_self_link.go.erb
+  DicomStore: !ruby/object:Overrides::Terraform::ResourceOverride
+    description: |
+      {{description}}
+    id_format: "{{dataset}}/dicomStores/{{name}}"
+    import_format: ["{{dataset}}/dicomStores/{{name}}"]
```

The import regex generator currently only matches fields that are in between slashes and doesn't do well with variables that contain slashes.
see https://github.com/terraform-providers/terraform-provider-google-beta/pull/858/files#diff-14199601f6d40245f2b47f254e2722deR217

We would usually set it to something like `"projects/{{project}}/locations/{{location}}/datasets/{{dataset}}/dicomStores/{{name}}"` but in this case there would need to be settable values for `project` and `location` which for these resources is not needed.

This is a long way of saying that magic modules doesn't currently have adequate import support for this type of resource without adding a `custom_import` method here. You are welcome to add custom import support if you need it or you can exclude import for now and wait until we fix https://github.com/GoogleCloudPlatform/magic-modules/issues/1946

```suggestion
    exclude_import: true
```